android/app/src/main/AndroidManifest.xml

```diff
@@ -124,6 +125,17 @@
 
             </intent-filter>
 
+            <intent-filter tools:ignore="AppLinkUrlError">
+                <action android:name="android.intent.action.VIEW" />
+                <action android:name="android.intent.action.EDIT" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <data
```

Comment:
just a heads-up, 
The path-matcher in android is very primitive and won't match paths with multiple `.`
Additional `<data>` sections will have to be added at some point to compensate.